app/src/main/java/com/gh4a/activities/IssueListActivity.java

```diff
@@ -330,6 +330,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 Intent intent = IssueActivity.makeIntent(this, mRepoOwner, mRepoName,
                         issue.getNumber());
                 startActivityForResult(intent, REQUEST_ISSUE_AFTER_CREATE);
+                // TODO: Handle this
```

Comment:
I'd also like to have IssueEditActivity launched as a separate task when you are creating a new one (not when editing). But I'm not sure how to solve communication back to here.